RecaptchaValidation/Services/RecaptchaService.cs

```diff
@@ -47,57 +44,51 @@ public async Task<IRecaptchaResponseMessage> Execute()
                 
                 if (!Response.success)
```

Comment:
This is not the same thing as `EnsureSuccessStatusCode()`

`EnsureSuccessStatusCode()` validates whether or not the return code was within the 200's range. 

`Response.Success` is a field determined by google recaptcha's verify API endpoint. This will return false for multiple different reasons, but it will return the message with a 200 return code:   
![image](https://user-images.githubusercontent.com/43724406/223191766-3aff3dad-f048-47ed-9c89-f383e80ebc27.png)
![image](https://user-images.githubusercontent.com/43724406/223191871-c9d1bba6-1bb6-4775-abd1-63867e5fb027.png)

Reply:
I realized this after I made the commit and didn't want to update my comment.

This indicates a pretty shitty API. Those should be `400 BAD REQUEST` response codes. Returning a `200 OK` with an Error payload is not great API design.

In any case, you have what you have to work with. Sounds good to me.